awx/ui/src/screens/Inventory/InventoryDetail/InventoryDetail.js

```diff
@@ -104,6 +125,22 @@ function InventoryDetail({ inventory }) {
             isEmpty={instanceGroups.length === 0}
           />
         )}
+        {prevent_instance_group_fallback && (
+          <Detail
+            label={t`Prevent Instance Group Fallback`}
+            dataCy="inv-detail-prevent-instnace-group-fallback"
+            helpText={helpText.preventInstanceGroupFallback}
+          />
+        )}
+        {renderOptionsField && (
+          <Detail
+            fullWidth
+            label={t`Enabled Options`}
+            value={renderOptions}
+            dataCy="jt-detail-enabled-options"
+            helpText={helpText.enabledOptions}
+          />
+        )}
```

Comment:
can these two conditionals be grouped into one, since `renderOptionsField === prevent_instance_group_fallback` (assigned on line 65)

Reply:
We could but I kept them separate to mimic how JTs work and to reduce work if we ever added another inventory option.